nitro-protocol/src/contract/state.ts

```diff
@@ -53,7 +55,12 @@ export interface VariablePart {
  * @returns the VariablePart, which usually changes during state channel updates
  */
 export function getVariablePart(state: State): VariablePart {
-  return {outcome: encodeOutcome(state.outcome), appData: encodeAppData(state.appData)};
+  return {
+    outcome: state.outcome,
+    appData: encodeAppData(state.appData),
```

Comment:
Outside the scope of this PR -- This seems a little strange, to me. I wouldn't expect we need to re-encode `appData` when extracting it from the state?

Reply:
If I remember correctly, this was added to be correspond to `go-nitro` state encoding.